### DISCUSSION TITLE
NSPanel Pro 120 support

### DISCUSSION THREAD

**Discussion by rwalker777** [Blackymas/NSPanel_HA_Blueprint#2248] [General]
Is it planned to add support for the NSPanel Pro 120?

**Comment by edwardtfn** [comment]
Nope.
NSPanel Pro is based on Android and cannot run ESPHome.

**Comment by Onepamopa** [comment]
So there's no way of converting Pro / Pro 120 to work?

**Reply by edwardtfn** [reply]
No. That is not possible, unfortunately.

**Reply by Onepamopa** [reply]
Has any effort been made to contact Sonoff in regards to making these displays more HA-friendly? By doing so, they stand to benefit from additional sales.

**Reply by plasterer41** [reply]
> Has any effort been made to contact Sonoff in regards to making these displays more HA-friendly? By doing so, they stand to benefit from additional sales.

you can install fully kiosk onto those and run it like a tablet/phone to display your dashboards

**Reply by Onepamopa** [reply]
I know, but if that's the case - why choose a sonoff display at all? Most of these displays are from their own ecosystems. HA doesn't have any "native" displays the way Tuya or Sonoff do. Especially the newer Tuya displays are, well... KILLERS.

**Reply by Onepamopa** [reply]
So... again - any plans to contact Sonoff in regards to making the pro / pro 120 more HA-friendly?

**Comment by Onepamopa** [comment]
Here's another interesting display based on ESP32: https://www.aliexpress.com/item/1005006335587633.html

**Reply by edwardtfn** [reply]
I have one of this... It works ok with ESPHome and LVGL, but not the TFT from this project, so you cannot use anything from this repo. 😩
The material looks more cheap plastic when compared to NSPanel, and no physical buttons.
